person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = nil
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
Please define a method for adding rentals in `Person` class. You may follow [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) link tp learn more about it.

Reply:
@SajjadAhmad14 no you mustn't there is a many-to-many relationship between the Person class and Book class so to associate them we defined an intermediate class Rental class so all methods to relate Person and Book are inside Rental class if you read my test script and run it you will realize when you call rental class the rentals array inside either Book or Person objects updated automatically